include/libc/stdint.h

```diff
@@ -1,8 +1,8 @@
 #ifndef STDINT_H
 #define STDINT_H
 
-typedef signed int intptr_t;
-typedef unsigned int uintptr_t;
+typedef signed long intptr_t;
+typedef unsigned long uintptr_t;
```

Comment:
Out of curiosity do we have a source for this change?

Reply:
I dug through standard headers provided by GCC to see what the default for `uintptr_t` is, and it uses `unsigned long` usually. It's also required for matching in some spots

Reply:
ftr my system has
```c
/* Types for `void *' pointers.  */
#if __WORDSIZE == 64
# ifndef __intptr_t_defined
typedef long int		intptr_t;
#  define __intptr_t_defined
# endif
typedef unsigned long int	uintptr_t;
#else
# ifndef __intptr_t_defined
typedef int			intptr_t;
#  define __intptr_t_defined
# endif
typedef unsigned int		uintptr_t;
#endif
```
(aka int for 32 bits, long for 64 bits)

but this really doesn't matter, we can choose whatever we want as long as it works for compiling in accordance to the C standard